test/new-e2e/pkg/e2e/suite_utils.go

```diff
@@ -20,3 +30,73 @@ func (tl testLogger) Write(p []byte) (n int, err error) {
 	tl.t.Log(string(p))
 	return len(p), nil
 }
+
+// GetRootOutputDir creates and returns a directory for tests to store output files and artifacts.
+// A timestamp is included in the path to distinguish between multiple runs, and os.MkdirTemp() is
+// used to avoid name collisions between parallel runs.
+//
+// A new directory is created on each call to this function, it is recommended to save this result
+// and use it for all tests in a run. For example see BaseSuite.GetTestOutputDir().
+//
+// See runner.GetProfile().GetOutputDir() for the root output directory selection logic.
+//
+// See GetTestOutputDir for a function that returns a subdirectory for a specific test.
+func GetRootOutputDir() (string, error) {
+	outputRoot, err := runner.GetProfile().GetOutputDir()
+	if err != nil {
+		return "", err
+	}
+	// Append timestamp to distinguish between multiple runs
+	// Format: YYYY-MM-DD_HH-MM-SS
+	// Use a custom timestamp format because Windows paths can't contain ':' characters
+	// and we don't need the timezone information.
+	timePart := time.Now().Format("2006-01-02_15-04-05")
+	// create root directory
+	err = os.MkdirAll(outputRoot, 0755)
+	if err != nil {
+		return "", err
+	}
+	// Create final output directory
+	// Use MkdirTemp to avoid name collisions between parallel runs
+	outputRoot, err = os.MkdirTemp(outputRoot, fmt.Sprintf("%s_*", timePart))
+	if err != nil {
+		return "", err
+	}
+	if os.Getenv("CI") == "" {
+		// Create a symlink to the latest run for user convenience
+		// TODO: Is there a standard "ci" vs "local" check?
+		//       This code used to be in localProfile.GetOutputDir()
+		latestLink := filepath.Join(filepath.Dir(outputRoot), "latest")
+		// Remove the symlink if it already exists
+		if _, err := os.Lstat(latestLink); err == nil {
+			err = os.Remove(latestLink)
+			if err != nil {
+				return "", err
+			}
+		}
+		err = os.Symlink(outputRoot, latestLink)
+		if err != nil {
+			return "", err
+		}
+	}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
What about passing the profile, and having the profile passing a path to the output root directory. Then each suite would create a directory inside the root directory, while the generic one is static.

Reply:
I'm not sure I'm understanding.
`profile.GetOutputDir()` already returns static output dir, e.g. `~/e2e-output`. The suite calls `GetRootOutputDir()` to create a new subdirectory for itself.

Reply:
The rename to `Create` fixes this too

Reply:
What I meant in my message is that is profile that knows where you are running. You might have a generic profile `GetLatestLink` that cleans up symlinks only on `localprofile` and that does nothing on ci profile. You would call it here with `runner.GetProfile().CleanupOldLinks()` or a better name.